docs/decisions/0004-add-access-policy-functionality.rst

```diff
@@ -270,6 +270,38 @@ then after that prioritizing subsidies with smaller balances.  Here’s a protot
      # Simply pick the first policy:
      return sorted_policies[0]
 
+Concurrency and Transaction Isolation
```

Comment:
Do we need to update this section(text + code) or I am missing something? Here we are describing multiple locking constraints. But in the actual implementation we are just locking on `policy uuid`.

Reply:
added learner and content ids to the locking calls ... to be used as needed by the proxy models